processing/src/main/java/io/druid/query/Druids.java

```diff
@@ -462,13 +463,9 @@ public TimeseriesQueryBuilder filters(String dimensionName, String value)
       return this;
     }
 
-    public TimeseriesQueryBuilder filters(String dimensionName, String value, String... values)
```

Comment:
IMO the old signature was better- it enforces at least one value must be passed in. an `IN ()` with empty list seems strange to me and not very meaningful.